### PR TITLE
fixed the sdk bug where one users middleware headers were copied to o…

### PR DESCRIPTION
**Pull request by grastogi23**
…ther session

### CODE REVIEW

File: python/avi/sdk/avi_api.py [hunk 1]
```diff
@@ -141,10 +142,11 @@ def __init__(self, controller_ip, username, password=None, token=None,
                 {"api": self, "last_used": datetime.utcnow()}
             user_session = self
         self.headers = copy.deepcopy(user_session.headers)
```
Comment (ypraveen) [inline python/avi/sdk/avi_api.py]: remove this line that is copying the headers?

Comment (grastogi23) [inline python/avi/sdk/avi_api.py]: done

**Comment by ypraveen** [comment]
I don't see the change

**Comment by grastogi23** [comment]
that was weird. I did a push but seems like it reverted the commits... now it shows up correctly

**Comment by ypraveen** [comment]
@nmandeyam check this

**Pull request merged**